file_entity.module

```diff
@@ -709,31 +647,6 @@ function file_entity_admin_menu_map() {
 }
 
 /**
- * Generates a token to protect a file download URL.
- *
- * This prevents unauthorized crawling of all file download URLs since the
- * {file_managed}.fid column is an auto-incrementing serial field and is easy
- * to guess or attempt many at once. This can be costly both in CPU time
- * and bandwidth.
- *
- * @see image_style_path_token()
- *
- * @param object $file
- *   A file entity object.
- *
- * @return string
- *   An eight-character token which can be used to protect file downloads
- *   against denial-of-service attacks.
- */
-function file_entity_get_download_token(File $file) {
```

Comment:
maybe we could just use the standard csrf token protection for this and drop this?